.github/workflows/e2e-ci.yaml

```diff
@@ -73,6 +73,62 @@ jobs:
         package: '${{ env.AZURE_FUNCTIONAPP_PACKAGE_PATH }}output'
         publish-profile: ${{ secrets.AZURE_FUNCTIONAPP_PUBLISH_PROFILE }}
 
+  build_push_docker_images:
+    needs:
+    - env_var
+    name: Build and Push Docker images
+    runs-on: ubuntu-latest
+    if: needs.env_var.outputs.RunE2ETestsOnly != 'true'
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout current branch
+      with:
+        fetch-depth: '2'
+
+    - run: |
+        if [ ${{ github.ref	}} = "refs/heads/dev" ]; then
+        echo "dev"
+        IMAGE_TAG="$DEV_IMAGE_TAG"
+        elif [ ${{ github.ref	}} = "refs/heads/master" ]; then
+        echo "master"
+        IMAGE_TAG="$MASTER_IMAGE_TAG"
+        elif [ ${{ github.event_name }} == "pull_request" ]; then
+        echo "pull request"
+        IMAGE_TAG="pr-${{ github.event.number}}"
+        else
+        echo "all others"
+        IMAGE_TAG="custom-${GITHUB_REF##*/}"
+        fi
+        echo "Using image tag prefix $IMAGE_TAG"
+        IMAGE_TAG="$IMAGE_TAG-${{ github.run_number }}"
+        echo "Using image tag $IMAGE_TAG"
+        echo "::set-env name=NET_SRV_VERSION::$IMAGE_TAG"
+        echo "::set-env name=LBS_VERSION::$IMAGE_TAG"
+      env:
+        ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+    - uses: docker/setup-buildx-action@v1
+      id: buildx
+      with:
+         install: true
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Build Docker LoRaWanNetworkServer images
+      run: |
+        echo ${{ secrets.ACR_CREDENTIALS }} | docker login "loramoduleintegrationtest.azurecr.io" --username  "loramoduleintegrationtest" --password-stdin
+        docker buildx bake --push LoRaWanNetworkServerarm32
+        # docker buildx bake --push LoraBasicsStationarm32v7
```

Comment:
Can we add a comment here that specifies that this docker image is not rebuilt every time as it's painful to do so? (time-wise)

Reply:
Done